packages/camera/camera_avfoundation/lib/src/avfoundation_camera.dart

```diff
@@ -552,6 +536,38 @@ class AVFoundationCamera extends CameraPlatform {
     return Texture(textureId: cameraId);
   }
 
+  String _serializeFocusMode(FocusMode mode) {
```

Comment:
I added these instead of continuing to use the ones from the platform interface package (the versions without the `_`) because having serialization in the platform interface package and deserialization in the implementation package is an anti-pattern since it means changes in one package can break internal communication in another. We created a lot of it by accident during the initial process of federating plugins; Pigeon migration has been eliminating it, but doing it now meant I could remove all the native code that required an "invalid" state for these enums, since I can guarantee within this package that we won't get a state at the native layer that we don't handle.

(These methods will become enum->enum conversion methods instead of enum->string conversion when I finish the Pigeon migration for this package.)